README.md

```diff
@@ -31,7 +31,8 @@ StatsD config file.
     secure: true,                 // OPTIONAL (boolean), whether or not to use secure protocol to connect to Instrumental, default true
     verifyCert: true,             // OPTIONAL (boolean), should we attempt to verify the server certificate before allowing communication, default true
     timeout: 10000,               // OPTIONAL (integer), number of milliseconds to wait for establishing a connection to Instrumental before giving up, default 10s
-    recordCounterRates: true      // OPTIONAL (boolean) whether or not to send ".rate" metrics with counters, default true
+    recordCounterRates: false     // OPTIONAL (boolean) whether or not to send ".rate" metrics with counters, default true
```

Comment:
No, the default is still true if you specify nothing.  We just recommend setting it to false.

This is confusing and maybe I should have left it out.